chat-app-express/app.js

```diff
@@ -38,32 +40,29 @@ app.use(function(req, res, next) {
   next();
 });
 
-// DB connetion
-mongoose.connect(process.env.DBLINK);
-
 io.on('connection', (socket) => {
```

Comment:
create a separate file for socket connection

Reply:
Done